Handle profit on genesis for eth validators (#5409)
rotki/rotki: docs/changelog.rst

* :bug:`-` Fixed an edge case where users would be swamped with "This socket is already used by another greenlet" errors during websocket communication between backend and frontend.
* :bug:`-` Transfers of ether between tracked accounts will now have a correct label in the UI.
* :bug:`-` Trades involving delisted bitfinex pairs will now be properly read by rotki.

> * :bug:`5390` Profit for ethereum validators will now be handled correctly when it was made in the first day after deposit.

Reviewer: Related to my other comment. This seems to only be for the genesis day. Not deposit.

```suggestion
* :bug:`5390` Profit for ethereum validators will now be handled correctly if the deposit was made on beacon chain genesis.
```
